src/Test/Perf/util/test_util.csx

```diff
@@ -111,6 +111,16 @@ string ReleaseVbcPath()
     return Path.Combine(BinReleaseDirectory(), "vbc.exe");
 }
 
+string GetCPCDirectoryPath()
+{
+    return Environment.ExpandEnvironmentVariables(@"%SYSTEMDRIVE%\CPC");
```

Comment:
I really really hate it when installers trash my root directory.  This should probably be `MyTempDirectory()`.